feat: expose supply pressure for GazBoilers

detzen opened the pull request:
Added getSupplyPressure() method for GazBoilers which is exposed for some time. 
See Issue #606  

I additionally added test data for my Vitodens 200W B2HF unit.

detzen committed:
added getSupplyPressure() for GAzBoilers

detzen committed:
added test data for Vitodens 200W B2HF

CFenner reviewed (commented):
Thanks for the PR, can you please add a short test file that references your test data and executes the new function: https://github.com/detzen/PyViCare/blob/0d23207afdef8b7e93a18be9154ea68e5d2fd694/tests/test_Vitodens200W.py#L1-L14

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

detzen committed:
added tests/test_Vitodens200W_B2HF.py

Added test file to execute new functions self.device.getSupplyPressure() and self.device.getSupplyPressureUnit() against test data in tests/response/Vitodens200W_B2HF.json

detzen commented:
I just added the requested test file which executes the new functions

CFenner committed:
Merge branch 'master' into feat/system_pressure_gaz_boiler

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_Vitodens200W_B2HF.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner committed:
Update test_TestForMissingProperties.py

CFenner commented on PyViCare/PyViCareGazBoiler.py:
```diff
@@ -260,6 +260,15 @@ def getPowerSummaryConsumptionDomesticHotWaterLastSevenDays(self):
+        return str(self.service.getProperty("heating.sensors.pressure.supply")["properties"]["value"]["unit"])
+
+    @handleNotSupported
+    def getSupplyPressure(self) -> float:
```
We definitely should have more typing!
